pom.xml

```
@@ -38,6 +38,11 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>2.0.6</version>
```

```suggestion
      <version>2.0.9</version>
```

2.0.9 would match https://github.com/ome/bioformats/pull/3956 and https://github.com/ome/ome-common-java/pull/73